iot-e2e-tests/common/src/main/java/com/microsoft/azure/sdk/iot/common/tests/serviceclient/RegistryManagerTests.java

```diff
@@ -374,7 +375,38 @@ public void apply_configuration_e2e() throws Exception
         registryManager.applyConfigurationContentOnDevice(deviceForTest, content);
     }
 
+    @Test
+    public void deviceCreationWithSecurityScope() throws IOException, InterruptedException, IotHubException
```

Comment:
Should we combine this with the basic crud test above?  seems like some redundancy here...

Reply:
sure thing

Reply:
Actually, since the registry manager ops aren't that expensive, I'd rather keep this test separate. It has some edge-specific terminology that I'd rather not mix into a vanilla Iot hub test